examples/braket_features/Simulating_Advanced_OpenQASM_Programs_with_the_Local_Simulator.ipynb

```diff
@@ -0,0 +1,726 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "# Simulating Advanced OpenQASM Programs with the Local Simulator\n",
+    "\n",
+    "The `LocalSimulator` now supports simulating OpenQASM programs and the scope\n",
+    "of supported language features is larger than previously offered on Braket!\n",
+    "\n",
+    "This notebook will demonstrate how to use many different OpenQASM features\n",
+    "with the `LocalSimulator`. For detailed documentation on language specification,\n",
+    "see https://openqasm.com/language/index.html."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Creating the Device\n",
+    "\n",
+    "We create the device as before."
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "outputs": [],
+   "source": [
+    "from braket.devices import LocalSimulator\n",
+    "\n",
+    "device = LocalSimulator()\n"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
```

Comment:
I think it shouldn't show anything related to PyCharm in the metadata, can you confirm if this aligns with out metadata structure for other notebooks.

Reply:
Fixed by importing notebook into an NBI